src/domUpdate.js

```diff
@@ -8,6 +8,10 @@ const domUpdate = {
     inputs.forEach(input => input.value = '')
   },
 
+  displayAlertMessage: (sleepInput, activityInput, hydrationInput) => {
```

Comment:
For reference: Doesn't display information on main page display after alert. May want to look into it if we have time